R/expect_length_linter.R

```diff
@@ -8,14 +8,18 @@
 #' @seealso [linters] for a complete list of linters available in lintr.
 #' @export
 expect_length_linter <- function() {
-  # TODO(michaelchirico): also catch expect_true(length(x) == 1)
-  xpath <- sprintf("//expr[
-    expr[1][SYMBOL_FUNCTION_CALL[text() = 'expect_equal' or text() = 'expect_identical']]
-    and expr[
-      expr[1][SYMBOL_FUNCTION_CALL[text() = 'length']]
-      and (position() = 2 or preceding-sibling::expr[NUM_CONST])
-    ]
-    and not(SYMBOL_SUB[text() = 'info' or contains(text(), 'label')])
+  xpath <- glue::glue("//expr[
+    expr[1][SYMBOL_FUNCTION_CALL[text() = 'expect_equal' or
```

Comment:
I'm surprised by grouping the `expect_equal`/`expect_identical` logic in the same condition, I would have expected that to be pretty separate from `expect_true` TBH.

PS, while you're at it, can you please remove the `//expr` condition here? #1358